server/models/Event.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const eventSchema = mongoose.Schema(
+  {
+    name: { type: String, required: true },
+    eventDate: { type: Date, required: true },
+    location: {
+      latitude: { type: Number },
+      longitude: { type: Number },
+      address: { type: String },
+    },
+    attendees: [{ type: mongoose.Schema.Types.ObjectId, ref: 'profile' }],
+    host: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'profile',
+    },
+  },
+  { timestamps: true },
+);
+
+module.exports = Event = mongoose.model('event', eventSchema);
```

Comment:
If your'e not using `Event` for anything else in this file, consider vomiting `Event` `module.exports  = mongoose.model('Event', eventSchema);`